From a8b51239fa8edba611b6f35df9a523cd79c07003 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Mon, 16 Jan 2012 13:18:17 +0100
Subject: [PATCH 214/217] logind: downgrade login message to debug

The messages make people nervous.

https://bugzilla.redhat.com/show_bug.cgi?id=727315
(cherry picked from commit 1637a8be5570dff7ce402451240b28ddb54e5dca)
---
 src/login/logind-user.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/login/logind-user.c b/src/login/logind-user.c
index 56c7de4..432d609 100644
--- a/src/login/logind-user.c
+++ b/src/login/logind-user.c
@@ -310,7 +310,7 @@ int user_start(User *u) {
         if (u->started)
                 return 0;
 
-        log_info("New user %s logged in.", u->name);
+        log_debug("New user %s logged in.", u->name);
 
         /* Make XDG_RUNTIME_DIR */
         r = user_mkdir_runtime_path(u);
@@ -425,7 +425,7 @@ int user_stop(User *u) {
         assert(u);
 
         if (u->started)
-                log_info("User %s logged out.", u->name);
+                log_debug("User %s logged out.", u->name);
 
         LIST_FOREACH(sessions_by_user, s, u->sessions) {
                 k = session_stop(s);
-- 
1.7.8.3

